<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilePage</class>
 <widget class="QWidget" name="FilePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Page</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="sidebarTree">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string>QTreeWidget {
    background-color: #f5f5f5;
    border: none;
    border-right: 1px solid #e0e0e0;
}
QTreeWidget::item {
    height: 32px;
    padding-left: 5px;
}
QTreeWidget::item:selected {
    background-color: #e3f2fd;
    color: #1976d2;
}</string>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="mainLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="toolbarFrame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
       <property name="styleSheet">
        <string>QFrame#toolbarFrame {
    background-color: white;
    border-bottom: 1px solid #e0e0e0;
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLineEdit" name="pathEdit">
          <property name="readOnly">
           <bool>true</bool>
          </property>
          <property name="placeholderText">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnImport">
          <property name="text">
           <string>导入</string>
          </property>
          <property name="icon">
           <iconset theme="document-open"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnExport">
          <property name="text">
           <string>导出</string>
          </property>
          <property name="icon">
           <iconset theme="document-save"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnNewFolder">
          <property name="text">
           <string>新建文件夹</string>
          </property>
          <property name="icon">
           <iconset theme="folder-new"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDelete">
          <property name="text">
           <string>删除</string>
          </property>
          <property name="icon">
           <iconset theme="edit-delete"/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRefresh">
          <property name="text">
           <string>刷新</string>
          </property>
          <property name="icon">
           <iconset theme="view-refresh"/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="fileList">
       <property name="styleSheet">
        <string>QTreeWidget {
    border: none;
    background-color: white;
}
QTreeWidget::item {
    height: 30px;
}
QTreeWidget::item:selected {
    background-color: #e3f2fd;
    color: black;
}</string>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="itemsExpandable">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <column>
        <property name="text">
         <string>名称</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>类型</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>大小</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>修改日期</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>